_docs/tasks/basic-access-control.md

```diff
@@ -89,6 +101,25 @@ Using a whitelist is a two step process.
    `checkExpression` is evaluated and checked against the list `[v1, v2]`. The check behavior can be changed to a blacklist by specifying
    `blacklist: true`. The expression evaluator returns the value of the `version` label as specified by the `checkExpression` key.
 
+The alpha version of `istioctl` does not yet support
+pushing adapter configurations, although there is a
+[workaround]({{home}}/docs/concepts/policy-and-control/mixer-aspect-config.html#pushing-configuration)
+if you want to try it.
```

Comment:
Mandar seemed to want to include the whitelist example, but the way he had it just left the reader hanging ... so how do I do it?  So I added this to explain it's not supported yet, but you can try the workaround if you want.